main.cpp

```diff
@@ -55,11 +61,50 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
+    int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    
+    int wheel = 3;
+    int window = 4;
+    int lights = 2;
+
+    char foo = 'a';
+    char bar = 'b';
+    char item = 'c';
+
+    float loudness1 = 1.0f;
+    float loudness2 = 0.5f;
+    float loudness3 = 0.3f;
+
+    double gallons = 1.1;
+    double litters = 0.4;
+    double ounces = 0.3;
+
+    bool sun = true;
+    bool rain = false;
+    bool snow = false;
     
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    ignoreUnused(wheel);
+    ignoreUnused(window);
+    ignoreUnused(lights);
+
+    ignoreUnused(foo);
+    ignoreUnused(bar);
+    ignoreUnused(item);
+
+    ignoreUnused(loudness1);
+    ignoreUnused(loudness2);
+    ignoreUnused(loudness3);
+
+    ignoreUnused(gallons);
+    ignoreUnused(litters);
+    ignoreUnused(ounces);
+
+    ignoreUnused(sun);
+    ignoreUnused(rain);
+    ignoreUnused(snow);
```

Comment:
```
b) at the end of the function, pass each variable to a single call of the ignoreUnused function. see line 71 for an example
```
a **single call**.  only call it once, pass all of them in the same function call.
- [ ] revise